packages/charts/src/Charts/SeverityLine/SeverityLine.js

```diff
@@ -0,0 +1,73 @@
+import './severity-line.scss';
+
+import React, { useEffect, useState } from 'react';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core/dist/js/components/Tooltip/index';
+
+import PropTypes from 'prop-types';
+
+const SeverityLine = ({ title, value, tooltipMessage, config }) => {
+
+    const [ severity, setSeverity ] = useState();
+    const [ isTruncated, setIsTruncated ] = useState(false);
+    const { width, height } = config;
+    const CHART_MAP = {
```

Comment:
If you don't like the conditional accessor you can still make this into a function (and move it above the component definition), that way it won't be re-assigned on each render

```JS
const chartMapper = (width, value) => {
  return {
    1: '1',
    2: (width / 3),
    3: (width * (2 / 3)),
    4: (width - 1)
  }[value]
};
```